Project.toml

```
@@ -19,10 +19,17 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
+[weakdeps]
+InverseFunctions = "3587e190-3f89-42d0-90ee-14403ec27112"
+
+[extensions]
+InverseFunctionsDistributionsExt = "InverseFunctions"
```

Following SpecialFunctions etc. it should be
```suggestion
DistributionsInverseFunctionsExt  = "InverseFunctions"
```